GCP Service Broker with external mysql (#116)
kyma-project/addons: addons/google-cloud-platform-service-broker-0.1.0/docs/broker/plans-details.md

## Service description

The `GCP Service Broker` service provides these plan names and descriptions:

| Plan Name           | Description |
|---------------------|-------------|
| `Default`           | Installs the GCP Service Broker with the default configuration. |
| `External database` | Allows to configure the external MySQL database. |

## Provisioning parameters

These are the provisioning parameters for the corresponding plans:

`Default:`

| Parameter Name | Display Name | Type | Description | Required | Default Value |
|----------------|---------|-----|-------------|:----------:|---------------|
| **secretName** | `GCP Secret name` | `string` | Defines the name of the Secret from which the Service Broker will try to fetch credentials. | YES |  |

`External database:`

| Parameter Name          | Display Name | Type | Description | Required | Default Value |
|-------------------------|---------|-----|-------------|:----------:|---------------|

> | **secretName**          | `GCP Secret name` | `string` | Defines the name of the Secret from which the Service Broker will try to fetch credentials. | YES |  |

Reviewer: ```suggestion
| **secretName**          | `GCP Secret name` | `string` | Defines the name of the Secret from which the Service Broker gets credentials. | YES |  |
```
